### PR TITLE
New hiring posts

### PR DESCRIPTION
**Pull request by anisval**

### CODE REVIEW

File: _posts/2022-02-01-engineering-product-design.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+author_url : author/alvaronistalvalbuena/
+author_avatar: alvaronistalvalbuena
+read_time : 4
+excerpt: At HolidayCheck, we are proud of building and improving products and features that make holidays better. Our users trust HolidayCheck and expect from us the best content, great deals, top-of-the-notch user experience and a stable and fast platform.  
+feature_image: posts/2022-02-02-engineering-product-design/team.jpg
+
+---
+At HolidayCheck, we are proud of building and improving products and features that make holidays better. Our users trust HolidayCheck and expect from us the best content, great deals, top-of-the-notch user experience and a stable and fast platform.  
+
+Our product teams are the strongest unit of our product organisation. They are typically formed by 3-5 developers, 1 designer and 1 product manager and work independently to create a great experience for our users. Our developers, designers and product managers are those who know how to fulfil our goals and vision best and work together towards making better holidays.
```
Comment (smolak) [inline _posts/2022-02-01-engineering-product-design.md:16]: `... and they work` - they missing

Comment (smolak) [reply _posts/2022-02-01-engineering-product-design.md:16]: `... fulfil our goals and vision best, and ` - comma missing

**Comment by smolak** [comment]
I haven't checked all, because I think you should parse this text through [Grammarly](https://app.grammarly.com/) first. It will help you a great deal!

File: _posts/2022-02-01-engineering-product-design.md (hunk 1 above)
Comment (TejashreeAher) [reply _posts/2022-02-01-engineering-product-design.md:16]: Our developers, designers and product managers are those who know best how to fulfil our goals and vision and twork together towards making better holidays.

File: _posts/2022-02-01-engineering-product-design.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* E-Commerce
+* Sales
+* Data/ML
+* Platform
+* Apps
+* IT Services
+
+## Traffic 
+The teams inside Traffic are responsible for bringing the users to our platform. Our projects include keywords like SEO, landing pages or user generated content. We have our hands and minds on all those pages where our users land at HolidayCheck when they start planning their holidays. The teams are all Fullstack, being Javascript, Typescript, React, NodeJS, GraphQL, ElasticSearch or Java some of the technologies they use. 
+
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:29]: Javascript, Typescript, React, NodeJS, GraphQL, ElasticSearch and Java being some of the technologies they use

File: _posts/2022-02-01-engineering-product-design.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Our latest successful projects include: 
+* We help our users know which location is the right one for them, like [Greece](https://www.holidaycheck.de/urlaub/griechenland) 
+* We keep them informed about the latest [travel restrictions](https://www.holidaycheck.de/sommerurlaub) 
+* We show them the first [prices](https://www.holidaycheck.de/fruehbucher), when they are still undecided
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:33]: We show them early bird deals  ?

File: _posts/2022-02-01-engineering-product-design.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Our latest successful projects include: 
+* We help our users know which location is the right one for them, like [Greece](https://www.holidaycheck.de/urlaub/griechenland) 
+* We keep them informed about the latest [travel restrictions](https://www.holidaycheck.de/sommerurlaub) 
+* We show them the first [prices](https://www.holidaycheck.de/fruehbucher), when they are still undecided  
+
+## E-Commerce 
+At the Ecom teams, the focus lies on helping users on our platform find the right hotel, the best offers and deals to have an unforgettable vacation. We continuously improve the booking funnel, develop services connecting to our external partners or develop the newest features related to the HolidayCheck Club. The teams are both Fullstack and Backend focused, and the technologies include: Javascript, React, NodeJS, Scala.
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:36]: In the Ecom teams

File: _posts/2022-02-01-engineering-product-design.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+* We help our users know which location is the right one for them, like [Greece](https://www.holidaycheck.de/urlaub/griechenland) 
+* We keep them informed about the latest [travel restrictions](https://www.holidaycheck.de/sommerurlaub) 
+* We show them the first [prices](https://www.holidaycheck.de/fruehbucher), when they are still undecided  
+
+## E-Commerce 
+At the Ecom teams, the focus lies on helping users on our platform find the right hotel, the best offers and deals to have an unforgettable vacation. We continuously improve the booking funnel, develop services connecting to our external partners or develop the newest features related to the HolidayCheck Club. The teams are both Fullstack and Backend focused, and the technologies include: Javascript, React, NodeJS, Scala. 
+
+Our greatest late additions are: 
+* Flex offers, where we are proud to have changed the travel industry by being the first ones to offer such a product 
+* New and optimised hotel pages, as [Dana Beach Resort](https://www.holidaycheck.de/hi/dana-beach-resort/1aa4c4ad-f9ea-3367-a163-8a3a6884d450)
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:40]: New and optimised hotel pages, eg. [Dana Beach Resort]

File: _posts/2022-02-01-engineering-product-design.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+Our greatest late additions are: 
+* Flex offers, where we are proud to have changed the travel industry by being the first ones to offer such a product 
+* New and optimised hotel pages, as [Dana Beach Resort](https://www.holidaycheck.de/hi/dana-beach-resort/1aa4c4ad-f9ea-3367-a163-8a3a6884d450) 
+* Premium offers and deals you can find all around the web 
+
+## Sales 
+At the Sales team we combine our massive reach and deep understanding of our markets to build innovative advertising products that generate the most effective and meaningful touchpoints between our Urlaubers and our leasure travel industry partners. We also make vacations better for our Urlaubers by developing tools to make relevant insights, benchmarks and reports easily accessible and actionable for our partners. Our team is full stack and we work with JavaScript, especially Node/Express and React.
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:44]: 'fullstack' just for consistency

File: _posts/2022-02-01-engineering-product-design.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+Our latest projects include: 
+* Building out the HolidayCheck Business Center for partners to navigate the HolidayCheck universe from one cockpit 
+* Making our advertising products more accessible and easier to use through self-booking 
+* Introducing auction/bidding to our advertising products for better price-discovery 
+
+## Data/ML 
+The Data and ML team is responsible for various projects, both on the platform and for generating internal insights. Our data lake is hosted on AWS and runs over 800 batch jobs every night, storing every offer that we see on our website and aggregating every interaction the user has with us. This allows us at HC to have all relevant data in one location and easily consumable. Based on this asset we have been building additional value with these projects:
```
Comment (TejashreeAher) [inline _posts/2022-02-01-engineering-product-design.md:52]: there are easily more than 1000 jobs

Comment (TejashreeAher) [reply _posts/2022-02-01-engineering-product-design.md:52]: "aggregating every interaction the user has with us" might be too specific again raising concerns on privacy. Can we change it to "processing the huge amounts of data giving customised experience to the user"

**Review by TejashreeAher — APPROVED** [review]

**Comment by SengitU** [comment]
Good luck with you search folks!

**Comment by smolak** [comment]
@SengitU - wow, nice to see you ;)
Yep, good luck finding great engineers, HC!

**Comment by anisval** [comment]
A very nice unexpected goody with this PR, getting to see good old faces :)

**Comment by anisval** [comment]
> I haven't checked all, because I think you should parse this text through [Grammarly](https://app.grammarly.com/) first. It will help you a great deal!

Loved that! New fav tool.

**Pull request merged**